clients/admin-ui/src/features/config-wizard/AuthenticateAwsForm.tsx

```diff
@@ -133,7 +133,9 @@ const AuthenticateAwsForm = () => {
               />
             ) : null}
 
-            {scannerError ? <ScannerError error={scannerError} /> : null}
+            {scannerError ? (
```

Comment:
Can this be simplified to `{scannerError && <ScannerError error={scannerError} isAWSScan />}`? This way you can avoid the `null` being returned

Reply:
I'm a fan of the `&&` but the pattern in Ctl has been the `: null` style. I'm hesitant to change it unless our linter includes a rule like [jsx-no-leaked-render](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-leaked-render.md) to catch the cases where `&&` is wrong.